### ISSUE TITLE
Quest for post office, post annex, post partner

### ISSUE THREAD

**Issue by georg-d** [streetcomplete/StreetComplete#2961]
### General
Affected tag(s) to be modified/added: https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dpost_office  and the 5/2021 approved "refining" tag https://wiki.openstreetmap.org/wiki/Key:post_office

**Question asked:** Is this a post office operated by _\<the postal service provider brand, e.g. "Deutsche Post">_ and offering the full range of postal services?

**Possible answers (table at the bottom may be more helpful at first contact)**

1. yes => adds _post_office=bureau_ and set _check_date_ 
2. Not operated by _\<the postal service provider brand>_ but offering (mostly) the full range of postal services - only provided by \<brand> => adds _post_office=post_annex_ and set _check_date_ 
3. Not operated by _\<the postal service provider brand>_ but offering (mostly) the full range of postal services - only or also by another brand than \<brand>  => adds _post_office=post_annex_ and set _check_date._ I do not know whether it's possible and matching StreetComplete's concept to trigger a follow-up quest to enter (or select from a list if well usable in the GUI despite the list's length) the 1..* values for _post_office:brand_ - I'd prefer it as the brand is usually a short string and it's usually much quicker & easier to see on location than for armchair mappers which postal service provider(s) a shop/office/... cooperates with.
3. Not operated by _\<the postal service provider brand>_ and offering only a limited range of postal services - only provided by \<brand> => adds _post_office=post_partner_ 
4. Not operated by _\<the postal service provider brand>_ and offering only a limited range of postal services of only or also another brand than \<brand> => adds _post_office=post_partner_ and set _check_date_ and, if possible, trigger a follow-up quest to enter the _post_office:brand._ IMHO _post_office:\<service>=*_ is as easy to map by armchair mappers (using the postal services provider's website) as on location (usually you need to ask the staff), so I don't have a strong preference for a follow-up quest, but if you have, feel free to comment & suggest.
5. The location is not any more offering postal services.

Please note I did by intention not add an answer for the combination _post_office=post_partner_ but no _amenity=post_office_ as I think this will make the quest too complex to solve in StreetComplete's style due to the many possible variations like e.g.: What shall be the future main key? Is the current brand the brand of the future main key or the postal services provider brand? Same for operator, ref, name etc. How to ask for the required and recommended tags associated with the future main key?

**I expect in some years, this quest will be extended** in the sense it will ask whether mapped _post_office=*_ information is still correct, i.e. whether a _post_office=bureau_ is still a _post_office=bureau_ or was turned into _post_office=post_annex,_ or whether a _post_office=post_partner_ is still partner of the mapped brand. Motivation for regularily re-checking is that the postal services market is in contant motion. Motivation to do not alreadey now is the low impact due to limited amout of existing _post_office=*._

### Checklist
Checklist for quest suggestions (see guidelines):
- [x] 🚧 To be added tag is established and has a useful purpose => ~143k uses of _amenity=post_office_ but only ~1k also carry _post_office=*_ so for the the vast majority (~142k), it's unclear whether they _in fact match OSM's definition_ of a post office due to years of lacking consensus on how to map post partners with a limited range of postal services. As a consequence, in a small sample I manually checked, a considerable share of kiosks, laundrys, etc were mapped and sometimes even rendered & listed as post offices despite not being one, which in turn causes people going there but not getting what they are looking for (e.g. stamps) because you can _only_ send + pick up parcels at these locations. 
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one) => quest adds the missing value for _post_office_ thus the quest won't re-appear
- [x] 🐿️ Easily answerable by everyone from the outside but a survey is necessary => nearly always, _post_office=bureau_ or not will be clear already from quite a distance, and often, _post_annex_ vs _post_partner_ becomes clear from the outside (e.g. for a small a 5m² kiosk or for a laundry with huge windows). The remaining may be known (local knoledge), skipped or clarified by asking.
- [x] 💤 Not an overwhelming percentage of elements have the same answer (No spam) => For many countries, I do expect no value combinations of post_office & brand to make up more than approcimately 25% of cases, and I do expect a considerable amount of locations to carry outdated tags, because several mappers reported they avoided to touch post office tagging at all and over years (due to missing tagging consensus).
- [x] 🕓 Applies to a reasonable number of elements (Worth the effort) => 143k objects

Besides the direct purpose, the quest creates the information base to build QA tools on top of, for example _post_office=post_partner_ without _post_office:\<service>=*_ could trigger further actions like maproulette quests or JSOM/vespucci warnings.

### Ideas for implementation
**Element selection:** _amenity=post_office_ and no value for key _post_office_ 

**Metadata needed:** AFAIK none. 

**Proposed GUI:** After looking at many pictures, I have the impression it's hard to find a photo that everyone will instantly understand: At the one hand, the post is red in several regions (UK etc) but yellow in several others (DE+CH+...) and blue in others (RUS), so whatever photo we choose, it will not be triggering associations for a considerable amount of users. At the other hand, the found photos are not clearly distinguishing the possible answers. 

Hence, I strongly tend to textual answers, maybe with a drawn picture if anyone skilled wants to contribute. Here a GUI sketch with example data:

| Is this a post office operated by DHL and offering the full range of postal services? |  |
| ------------- | ------------- |
| yes |  |
| Not operated by DHL but offering (mostly) the *full range* of postal services -  |  only provided by DHL |
|  |  only or also another brand than DHL |
| Not operated by DHL and offering only *a limited range* of postal services -  |  only provided by DHL |
|  |  only or also another brand than DHL |
| The location is not any more offering postal services. |  |
| Other answers... |  |

**Comment by mnalis** [comment]
I think your first checklist (for _to be added tag_) should note there is currently about 777 uses of [post_office=*](https://taginfo.openstreetmap.org/keys/?key=post_office#overview) key. And you can add `x` between square brackets like `[x]` to indicate checklist item as OK.

As for the locations to pick up / send items; they also might be [vending=parcel_pickup](https://wiki.openstreetmap.org/wiki/Tag:vending%3Dparcel_pickup) / [vending=parcel_mail_in](https://wiki.openstreetmap.org/wiki/Tag:vending%3Dparcel_mail_in))

**Comment by smichel17** [comment]
This would be Germany-only? Or exclude the US? I don't think the quest would make sense here; there are UPS/FEDEX/Amazon stores, but I would only call something a post office if it was operated by the US Postal Service.

**Comment by georg-d** [comment]
This quest would be in general globally with some few regions excluded e.g. if they only have "classic post offices".

Concerning the key & value names, they will not be visible in the quest's GUI (I intentionally designed it like that to avoid exactly reactions like yours) and please do not start a discussion here in the quest proposal, because we already had several lengthy discussions about the wording; result in a nutshell: Exactly _because_ few people consider "post office" a proper term for e.g. a pure Amazon parcel pickup/return location, existing key _post_office:type=*_ (12k uses) was renamed to _post_office=*_ and given a reworked meaning so we can now clearly distinguish between both types of locations. Concerning names, the community was unable to reach consensus on another term; essentially all proposed ones were valued equally bad. For more information, please see mailing list, proposals, forms etc.

**Comment by westnordost** [comment]
How would the surveyor know if "the full range of services" is provided? How would the surveyor know if it is operated by DHL/deutsche post itself or some other?

From my experience (=hopefully user point of view), there are just two types of things here:
1. proper post office
2. some other shop that also offers postal services

Is the tagging eligible to dumb it down to such a boolean question?

**Comment by westnordost** [comment]
No feedback, closing.
